checksums.json

```diff
@@ -1,147 +0,0 @@
-{
```

Comment:
It's not added to this directory though - it's only created inside the `puppetlabs-ciscopuppet-1.2.3.tar.gz` file. I just checked this by rerunning `puppet module build` and the file is not regenerated in this repo. We already exclude the build directory (`/pkg`), so the only way this would creep back in would be for you to unpack the module into this directory.